test/env.jl

```diff
@@ -181,3 +181,12 @@ end
 for (k, v) in pairs(original_env)
     ENV[k] = v
 end
+
+@testset "username()" begin
+    if Sys.isunix()
+        passwd = Libc.getpwuid(Libc.getuid())
+        @test username() == passwd.username
+    else
+        @test !isempty(username())
+    end
```

Comment:
Isn't there an environment variable one could compare to on Windows?

Reply:
Apparently, `USERNAME` should work. Although I am not sure if it always coincides with that returned by `GetUsername`.